### ISSUE TITLE
Verify that scientific notation uses multiplication symbol

### ISSUE THREAD

**Issue by arouinfar** [phetsims/inverse-square-law-common#50]
In discussing accessible scientific notation in https://github.com/phetsims/a11y-research/issues/105, I noticed that the multiplication symbol in ISLC sims appears to be the letter X. 

![image](https://user-images.githubusercontent.com/8419308/40811767-cc3f0952-64ef-11e8-9cd1-c82d6202a01e.png)
![image](https://user-images.githubusercontent.com/8419308/40812089-44fbaee4-64f1-11e8-9a1b-b9971f11490b.png)

We should be using the multiplication symbol which looks like (but is not actually) an X, see Area Model Multiplication for an example.

![image](https://user-images.githubusercontent.com/8419308/40811794-f14a3f8c-64ef-11e8-8415-7414becf4811.png)

**Comment by jessegreenberg** [comment]
Thanks @arouinfar, good catch. Should probably be using MathSymbols.TIMES now. Removing my assignment.
